tools/authors/authorxml.php

```diff
@@ -32,7 +32,8 @@
 header("Content-Type: text/xml; charset=$charset");
 echo "<?xml version=\"1.0\" encoding=\"$charset\" ?>\n";
 
-$result = mysqli_query(DPDatabase::get_connection(), "SELECT * FROM authors $clause");
+$sql = sprintf("SELECT * FROM authors %s", $clause);
```

Comment:
You don't need the sprintf() here since we aren't enforcing formatting. The original with the embedded $clause is fine.